doc/index.rst

```diff
@@ -91,7 +108,7 @@ Prerequisite
 In order for your repository to get checked with configurations in `industrial_ci`, it needs:
 
 * To be a `Catkin package <http://wiki.ros.org/ROS/Tutorials/catkin/CreatingPackage>`_ (uses CMake for build configuration), since many checks are triggered by the `Catkin`-based commands.
-* Build-able on Linux (as of Dec 2015, Ubuntu 14.04/Trusty is used). Although your repository is not necessarilly intended for Linux, checks are run on Linux.
+* Docker image for the targeted operating system is available on a Docker registry (e.g. http://hub.docker.com, Gitlab registry) either publicly or privately.
```

Comment:
Actually, this might confuse users if they are not used to Docker.
It is a requirement, but there is no special action to take for the ROS-supported linux systems.